sources/web/datalab/polymer/components/datalab-sidebar/datalab-sidebar.html

```diff
@@ -68,7 +68,7 @@
       .iron-selected > .sidebar-button > * {
         color: var(--selection-fg-color);
       }
-      @media only screen and (max-width: 970px) {
+      @media only screen and (max-width: 800px) {
```

Comment:
Assuming this is supposed to happen at the same width as the javascript code reference to the same number, is there a way this number can be in one place and referenced by both html and javascript?